lib/stitches/error.rb

```diff
@@ -1,9 +1,9 @@
 module Stitches
   class Error
     attr_reader :code, :message
-    def initialize(code:, message:)
-      @code    = code
-      @message = message
+    def initialize(options = {})
+      @code    = options[:code]
+      @message = options[:message]
```

Comment:
Good call.  Should these be `fetch`?  We don't have a test for it, but the way the kwargs would work is if you omitted either `code` or `message`, you'd get a runtime error from Ruby.  `fetch` would simulate that reasonably well.